File: 1436-destination-city/1436-destination-city.java
class Solution {
    public String destCity(List<List<String>> paths) {
        List<String> strings=new ArrayList<>();
        String a="";
        for(int i=0;i<paths.size();i++){
            strings.add( paths.get(i).get(0));
        }
        for(int i=0;i<paths.size();i++){
            if(!strings.contains(paths.get(i).get(1))){
                a=paths.get(i).get(1);
            }
        }
        return a;
    }
}

File: 1436-destination-city/1436-destination-city.cpp
class Solution {
public:
    string destCity(vector<vector<string>> &paths) {
        list <string>list1;
        for(int i=0;i<paths.size();i++){
            string dest=paths[i][1];
            int test=0;
            for(int j=0;j<paths.size();j++){
                if(paths[j][0]==dest)
                    test=1;
            }
            if(test!=1){
                return dest;
            }
            
        }
        return " ";
    }
};